### DISCUSSION TITLE
What to do when all songs in playlist have finished

### DISCUSSION THREAD

**Discussion by 1hitsong** [jellyfin-archive/jellyfin-roku-legacy#1652] [General]
Discussion continued from https://github.com/jellyfin/jellyfin-roku/pull/1644

**Comment by cewert** [comment]
@1hitsong would you like to continue this discussion? https://github.com/jellyfin/jellyfin-roku/pull/1644#issuecomment-1890619820

**Reply by 1hitsong** [reply]
No, you can close.
